OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -76,6 +76,9 @@ public class MissileInfo : IProjectileInfo
 		[Desc("Vertical rate of turn.")]
 		public readonly int VerticalRateOfTurn = 6;
 
+		[Desc("Gravity applied while in free fall.")]
```

Comment:
Siding with @reaperrr here. Gravity was falling in TS/RA2 as well. The description is fine.